futures-util/src/try_future/into_future.rs

```diff
@@ -0,0 +1,28 @@
+use core::mem::PinMut;
+use futures_core::{Future, TryFuture};
+use futures_core::task::{Context, Poll};
+
+/// Converts a `TryFuture` into a normal `Future`
+#[derive(Debug)]
+#[must_use = "futures do nothing unless polled"]
+pub struct IntoFuture<Fut> {
```

Comment:
This is a new invention. I think that could be useful. I originally planned to use it for something, but my plan changed.